src/test/java/teammates/test/cases/logic/FeedbackResponseCommentsLogicTest.java

```diff
@@ -93,7 +93,7 @@ public void testCreateFeedbackResponseComment() throws Exception {
         assertNotNull(actualFrComment);
 
         //delete afterwards
-        frcLogic.deleteFeedbackResponseComment(frComment);
+        frcLogic.deleteFeedbackResponseCommentById(frComment.getId());
```

Comment:
Are you sure that `frComment.getId()` return an real ID?

Reply:
Yes, this is a real Id. I verified this using `verifyPresentInDatastore` and `verifyAbsentInDatastore` before and after this execution. I will add `verifyAbsentInDatastore` after this to be sure.

Reply:
Can just use `frcLogic.getFeedbackResponseComment()` to verify.

Reply:
Wait, In L73, the ID is set to `null`.

Reply:
Actually there is a problem with the test. I'll create a separate issue for it. Currently can use `frComment.getId()`

Reply:
ok